blocks/callout/callout.js

```diff
@@ -0,0 +1,69 @@
+export default async function decorate(block) {
+  const calloutData = {
+    title: block.children[0].children[0].textContent.trim(),
+    description: block.children[0].children[1].children,
+    url: block.children[1].children[0].textContent.trim(),
+    buttonLabel: block.children[1].children[1].textContent.trim(),
+    altText: block.children[1].children[2].textContent.trim(),
+  };
+
+  let calloutBlock;
+  let calloutButton = null;
+
+  // Create container element
+  const calloutContainer = document.createElement('div');
+  calloutContainer.classList = 'callout-container';
+
+  // Populate callout content
+  const calloutContent = document.createElement('div');
+  calloutContent.classList = 'callout__content';
```

Comment:
it doesn't quite matter in this situation, but I've been using `classList.add()` because it doesn't overwrite any classes already on the component

Reply:
I'll update for consistency!